packages/eslint-plugin/src/configs/all.json

```diff
@@ -0,0 +1,49 @@
+{
+  "extends": "./configs/base.json",
+  "rules": {
+    "@typescript-eslint/adjacent-overload-signatures": "error",
+    "@typescript-eslint/array-type": "error",
+    "@typescript-eslint/ban-ts-ignore": "warn",
```

Comment:
> defaulting to `warn` just, because if the rule isn't recommended, it might be not an error, which this rule covers.

Hi, I strongly recommend to defaulting to "error".
This is the classic behaviour of `all` configs.

See 
- eslint : https://github.com/eslint/eslint/blob/master/conf/eslint-all.js
- tslint : https://github.com/palantir/tslint/blob/master/src/configs/all.ts
- react plugin : https://github.com/yannickcr/eslint-plugin-react/blob/master/index.js

Reply:
Thank you for pointing this out, all.json will have "error" for the rules.